impl/client/src/main/javascript/web/dojo/pentaho/common/MessageBox.js

```diff
@@ -21,6 +21,8 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_Templated", "dojo/on"
      {
         buttons: ['btn1','btn2','btn3'],
         messageType: null,                         // options are null, ERROR, WARN, INFO
+        responsive: true, // Not intended to be set to false.
+        responsiveClasses: "dw-sm",
```

Comment:
license header

Reply:
Just added the LICENSE.txt file. Thank you :)